modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -31,7 +31,7 @@ image::builtin-dashboard-selection-interaction.png[Built-in Dashboard Selector]
 
```

Comment:
Because I am swamped, I edited the white and green texts only.
If you need me to look at the blue later, please create a JIRA to remind yourself.

Reply:
I think this whole topic needs attention. I will assess it more critically when this is dropped. I might set up some time to look this over from an IA perspective, as it is getting very bulky.